pandora-server-directory/src/account/account.ts

```diff
@@ -89,8 +95,7 @@ export class Account {
 	public getAccountPublicInfo(): AccountPublicInfo {
 		return {
 			id: this.data.id,
-			// TODO: Allow choosing custom display name instead of always showing a username
-			displayName: this.data.username,
+			displayName: this.displayName,
```

Comment:
So I assume this means that the client no longer gets the full data package anywhere in pandora code that includes the username, but now only gets the display name from the database?

What is the default value for the display name on a new account? 
I assume it is maybe simply the account number?
Is there also a way to set this display name somewhere, like under settings > account?
Will we allow users to change the display name at any point in time and how often they want or can they only set one once?

Reply:
Sadly no, the client still gets the username with relationships (contracts, blocks), and that is a bit more complex to remove, since a copy of the username is already stored in the database

I just basically moved the todo to a getter property in the account that returns the username the same way as before

Reply:
Sekkmer opened a new issue: https://github.com/Project-Pandora-Game/pandora/issues/505